python/ray/tune/integration/pytorch_lightning.py

```diff
@@ -77,6 +77,14 @@ class TuneReportCallback(TuneCallback):
 
     Reports metrics to Ray Tune.
 
+    .. note::
```

Comment:
I thought we were not deprecating the tune integrations?

Reply:
Oh I see. Probably we just recommend the users to use LightningTrainer?

Reply:
Yeah we can recommend but I thought we were not deprecating

Reply:
Ok got it! I've updated the docstring!